cl/recap/mergers.py

```diff
@@ -1552,7 +1567,11 @@ async def merge_attachment_page_data(
         # we got the real value by measuring.
         if rd.page_count is None:
             rd.page_count = attachment["page_count"]
-        if rd.file_size is None and attachment.get("file_size_str", None):
+        # If we have file_size_bytes it should have max precision.
+        file_size_bytes = attachment.get("file_size_bytes")
+        if file_size_bytes is not None:
+            rd.file_size = file_size_bytes
```

Comment:
Does the file size is only for attachments?  I noticed it can be available for the main document too according to Juriscraper, but is not being added to the main document.

Reply:
yeah, although that's been an issue previously from the looks of it, might require some refactoring to fix that so I think best to just leave it as a follow up